specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/AvailableWorkloadProfiles.json

```diff
@@ -0,0 +1,133 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-06-01-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.App/locations/{location}/availableManagedEnvironmentsWorkloadProfileTypes": {
+      "get": {
+        "tags": [
+          "AvailableWorkloadProfiles"
+        ],
+        "summary": "Get available workload profiles by location.",
+        "description": "Get all available workload profiles for a location.",
+        "operationId": "AvailableWorkloadProfiles_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AvailableWorkloadProfilesCollection"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
```

Comment:
>./CommonDefinitions.json#/definitions/DefaultErrorResponse [](http://example.com/codeflow?start=23&length=58)

Any reason you aren't also getting this from v3 common types (ErrorResponse definition)?

Reply:
@mentat9, All of the ARM reviewed endpoints for our 2022-06-01-preview use the /CommonDefinitions.json#/definitions/DefaultErrorResponse as the default response. I did it too to keep it consistent. 

Any reason I should deviate and change it to common-types/resource-management/v3/types.json#/definitions/ErrorResponse ?

Reply:
It's not a requirement, but definitely recommended. We are trying to get RPs to use the common types as much as possible. It allows readers to know you have decided to follow the standard. Otherwise, it's hard to be sure without a painstaking comparison of your RP's "common" definitions with the common types. Since I don't have your branch, I don't even see files that aren't present in your PR, so I can't even see what the definition is. If your definition doesn't already match the v3 common type, we would definitely recommend switching.